SPECS/mariner-repos/mariner-repos.spec

```diff
@@ -30,16 +31,29 @@ Requires:   %{name} = %{version}-%{release}
 %description preview
 %{summary}
 
+%package extras
+Summary:  CBL-Mariner extras repository.
+Group:    System Envrionment/Base
+Requires: %{name} = %{version}-%{release}
+
+%description extras
+%{summary}
+
 %install
 rm -rf $RPM_BUILD_ROOT
-install -d -m 755 $RPM_BUILD_ROOT/etc/yum.repos.d
-install -m 644 %{SOURCE2} $RPM_BUILD_ROOT/etc/yum.repos.d
-install -m 644 %{SOURCE3} $RPM_BUILD_ROOT/etc/yum.repos.d
-install -m 644 %{SOURCE4} $RPM_BUILD_ROOT/etc/yum.repos.d
+%define repos_directory $RPM_BUILD_ROOT/etc/yum.repos.d
```

Comment:
In most specs we move the `%define` macros to the top of the file. I'd suggest either doing that or simply switching to bash variables, since we're only using this macro inside the `%install` section.